### ISSUE TITLE
{Freeway-Bug}[AdaptiveCards- Designer - Elements Properties]: Ensures attributes that begin with aria- are valid ARIA attributes.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8175]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL in browser.
2. Navigate to 'Element Properties' button press enter key to expand it.
3. Run the Accessibility Insight For Web tool and observe the issue.

**Actual Result:**
Attributes that begin with aria- are not valid ARIA attributes as on navigating to the button beside the edit controls it is announcing as **'Button Data Binding.'**.

**Expected Result:**
Attribute that begin with aria should have valid Aria attributes and it should be unique. Screen Reader should announce the unique name for all the buttons. It should announce as **'Data biding for Data context button'.** and on again navigating it should announce as **'Data binding for Only show when button'**. For each button unique aria label should be defined.

**Element path:** 
.default-ac-columnSet:nth-child(3) > .default-ac-container:nth-child(5) > div > div:nth-child(1) > .default-ac-actionSet > .default-ac-pushButton.default-style-default.default-ac-selectable

**Snippet:** 
<button type="button" class="default-ac-pushButton style-default default-style-default default-ac-selectable" tabindex="0" role="button" aria-label="..." title="Data Binding" aria-description="Data Binding" style="display: flex; align-items: center; justify-content: center; flex: 0 1 auto;">

**How to fix:** 
Fix any of the following:
  Invalid ARIA attribute name: aria-description

**User Impact:**
It will impact the screen reader if the valid aria attributes are not used as user will not be able to differentiate between two controls.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B507B5D90-F1B9-4AE1-91D1-2F1F2D6C7C1D%7D&file=MAS%204.1.2%20%E2%80%93%20Name%2C%20Role%2C%20Value.docx&action=default&mobileredirect=true

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

![MAS4 1 2-Same name is defined for the buttons](https://user-images.githubusercontent.com/95913748/208680762-f470ebe3-c275-438b-86a9-1293dd6de732.png)
![MAS4 1 2-Ensure attribute that begin with aria are valid ARIA attribute](https://user-images.githubusercontent.com/95913748/208680771-3441d9f0-40a4-4c22-ae8c-7b97afd5528f.png)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro on the below environment, hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25330.1000)
Browser Version: 114.0.1807.1
